web-components/src/components/phone-input/PhoneInput.ts

```diff
@@ -127,16 +139,50 @@ export namespace PhoneInput {
       this.formattedValue = new AsYouType(this.countryCode).input(input);
     }
 
+    handleCountryFocusOut({ detail }: any) {
+      const test = this.combobox!.shadowRoot!.querySelector(".group");
+      console.log(test);
+      console.log(this.countryCode);
```

Comment:
more console logs :)

also would make sure that this const gets removed, doesn't seem to be doing anything else.